### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by av-lukas**

### CODE REVIEW

File: endpoints/lookupVoterEmail/app.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // Note: /opt/Common is where all the lib layer code gets put
-//const { ApiResponse } = require("../../lib/ApiResponse");
-//const { Election } = require("../../lib/Election");
+// const { ApiResponse } = require("./lib/ApiResponse");
+// const { Election } = require("./lib/Election");
+// const { Voter } = require("./lib/Voter");
+// const { ApiRequire } = require("./lib/ApiRequire");
```
Comment (av-michael) [inline endpoints/lookupVoterEmail/app.js:5]: Should this be here?

File: README.md [hunk 2]
```diff
@@ -1,187 +1,25 @@
-# Change Log
+# Election Provisioner & Voter Registry
```
Comment (av-michael) [inline README.md:1]: Is it intentional that the previous readme is deleted?

**Review by av-michael — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,187 +1,25 @@
...
 
-Version 1.3.0
-* yearOfBirth added to voter specfication; returned in queries
-* Test data update SSN removed; DL and IDN entries updated
-* No current election typo error fixed
-* VIDN replaces IDNumber in lookupVoterEmail
-* "0" attribute values are now preserved in voter data model
-* Updates Affidavit file
+```bash
+$ curl -XPOST http://0.0.0.0:3009/setCurrentElection -H 'Content-Type: application/json' -d '{"electionId": "0c7a901a-e8fe-4601-8417-5d0823159d42"}'
```
Comment (av-mads) [inline README.md:18]: Where does this electionId come from?

File: README.md (hunk 2 above)
Comment (av-lukas) [reply README.md:1]: Yes. I found it to be mostly useless so I reduced it to what we care about.

File: db-init.sh [hunk 4]
```diff
@@ -23,4 +25,4 @@ aws dynamodb batch-write-item --request-items file://db-json/populate-voter-3.js
 aws dynamodb batch-write-item --request-items file://db-json/populate-application.json --endpoint-url http://localhost:8000  --output json --no-paginate
 
 
-aws dynamodb scan --table-name abc_voter_sessions_local --endpoint-url http://localhost:8000  --output json --no-paginate
\ No newline at end of file
+aws dynamodb scan --table-name abc_voter_sessions_local --endpoint-url http://localhost:8000  --output json --no-paginate
```
Comment (av-mads) [inline db-init.sh:28]: Db is running on port 8000 but the application is running on 3009, is that correct?

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+RUN curl -fsSL https://deb.nodesource.com/setup_$NODE_VERSION | bash -
+RUN apt-get install -y nodejs
+
+WORKDIR /usr/src/sam
+
+RUN curl -LO https://github.com/aws/aws-sam-cli/releases/download/$SAM_VERSION/aws-sam-cli-linux-x86_64.zip
+RUN sha256sum aws-sam-cli-linux-x86_64.zip
+RUN unzip aws-sam-cli-linux-x86_64.zip -d sam
+RUN ./sam/install
```
Comment (av-mads) [inline Dockerfile]: I dont know what's going on here, but I dont think I need to.

File: db-init.sh [hunk 6]
```diff
@@ -23,4 +25,4 @@ aws dynamodb batch-write-item --request-items file://db-json/populate-voter-3.js
 aws dynamodb batch-write-item --request-items file://db-json/populate-application.json --endpoint-url http://localhost:8000  --output json --no-paginate
```
Comment (av-mads) [inline db-init.sh:25]: Is this the file we are populating the db with?

File: README.md (hunk 3 above)
Comment (av-lukas) [reply README.md:18]: From the seeds. It's the only election available after seeding.

File: db-init.sh (hunk 4 above)
Comment (av-lukas) [reply db-init.sh:28]: Correct.

File: Dockerfile (hunk 5 above)
Comment (av-lukas) [reply Dockerfile]: 🤣

File: db-init.sh (hunk 6 above)
Comment (av-lukas) [reply db-init.sh:25]: Correct.

File: endpoints/lookupVoterEmail/app.js (hunk 1 above)
Comment (av-lukas) [reply endpoints/lookupVoterEmail/app.js]: No.

File: README.md (hunk 3 above)
Comment (av-mads) [reply README.md:18]: So it's the OSET election id?

Comment (av-lukas) [reply README.md:18]: I assume so, as it's the only election available in the DB.

**Pull request merged**